packages/forms-web-app/src/views/manage-appeals/your-appeals.njk

```diff
@@ -40,7 +40,7 @@
                   <a href="{{appealDetailsLink}}/{{item.caseReferenceSlug}}" class="govuk-link">{{item.caseReference}}</a>
                 </td>
                 <td class="govuk-table__cell">{{item.LPAApplicationReference}}</td>
-                <td class="govuk-table__cell"></td>
+                <td class="govuk-table__cell">{{item.questionnaireDueDate | govukDate('d MMMM yyyy HH:mm:ss') }}</td>
```

Comment:
this is displaying hours minutes and seconds - do we want this?

Reply:
ah sorry left over from testing